src/DotNetWorker/Configuration/WorkerOptions.cs

```diff
@@ -1,29 +1,27 @@
-// Copyright (c) .NET Foundation. All rights reserved.
+﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
-using System.Collections.Generic;
+using System.Text.Json;
+using Azure.Core.Serialization;
 
-namespace Microsoft.Azure.Functions.Worker.Configuration
+namespace Microsoft.Azure.Functions.Worker
 {
-    // TODO: Slim this down.
     public class WorkerOptions
     {
         public WorkerOptions()
         {
-            Capabilities = new List<string>();
+            var serializerOptions = new JsonSerializerOptions
```

Comment:
Is there a reason we're setting this? Expectations from our internal protocol?

Reply:
I debated this one -- I don't think anything internally matters here. But I was trying to pick the biggest difference between Json.NET and System.Text.Json. In retrospect, it makes sense (and it's easier to explain) if we just use the default here.